docs/.vuepress/config.js

```diff
@@ -77,7 +77,38 @@ module.exports = {
   },
   extendMarkdown: md => {
     md.use(require('markdown-it-attrs'))
+    const r = md.renderer.rules
+
+    const buildTag = (token, slf, classname) => {
+      // We need to add our md classes
+      const mdClasses = token.attrGet('class') || ''
+      token.attrSet('class', classname + ' ' + mdClasses)
+      let attrs = slf.renderAttrs(token)
+      return `<${token.tag} ${attrs}>`
+    }
+
+    r.heading_open = (tokens, idx, options, env, slf) => {
+      tkn = tokens[idx]
+      const anchor =
+        tkn.tag == 'h2' || tkn.tag == 'h3' ? 'markdown-heading' : ''
+      const classes = [
+        // text level class, h1 = l1, h2 = l2, etc
+        'l' + tkn.tag.substr(-1),
+        anchor,
+        // Add classes here
+        'tc-text-500'
+      ].join(' ')
+      return buildTag(tkn, slf, classes)
+    }
+    r.paragraph_open = (tokens, idx, options, env, slf) =>
+      buildTag(tokens[idx], slf, 'l7 tc-text300')
+    r.paragraph_close = () => '</p>'
```

Comment:
Why do you need to close the paragraph but nothing else?

Reply:
For some reason , not including the close tag will mean it doesn't render. Super weird little bug, I should probably report it to markdown-it